src/components/fields/multi-select/multi-select.tsx

```diff
@@ -28,7 +28,17 @@ export const MultiSelect = (props: IGenericFieldProps<IMultiSelectSchema>) => {
 	// EFFECTS
 	// =============================================================================
 	useEffect(() => {
-		setFieldValidationConfig(id, Yup.array().of(Yup.string()), validation);
+		setFieldValidationConfig(
+			id,
+			Yup.array()
+				.of(Yup.string())
+				.test("is-empty-array", "An option is required", (value) => {
```

Comment:
same as checkbox-group
- derive error message from required validation config
- this is only applicable if field is required